Classes/Extensions/DDDispatchQueueLogFormatter.m

```diff
@@ -93,48 +117,54 @@ - (void)setReplacementString:(NSString *)shortLabel forQueueLabel:(NSString *)lo
 #pragma mark DDLogFormatter
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-- (NSString *)stringFromDate:(NSDate *)date {
-    int32_t loggerCount = OSAtomicAdd32(1, &_atomicLoggerCount);
+- (NSDateFormatter *)createDateFormatter {
+    NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+    [self configureDateFormatter:formatter];
+    return formatter;
+}
```

Comment:
When created in "shareable" mode (meaning usable with multiple loggers) we'll have a dedicated NSDateFormatter instance per each thread where log formatter is used;
so yes, we may need several instances